lib/pedicel/base.rb

```diff
@@ -1,20 +1,24 @@
+# frozen_string_literal: true
+
 require 'openssl'
 require 'base64'
 
 module Pedicel
   class Base
-    SUPPORTED_VERSIONS = [:EC_v1]
+    SUPPORTED_VERSIONS = ['EC_v1']
 
-    def initialize(token, now: Time.now)
-      @token = token
+    def initialize(token, now: Time.now, config: Pedicel.config)
+      @token  = token
+      @config = config
     end
 
     def validate_content(now: Time.now)
-      raise VersionError, "unsupported version: #{version}" unless SUPPORTED_VERSIONS.include?(@token['version'])
-
-      raise ReplayAttackError, "token signature time indicates a replay attack (age #{now-cms_signing_time})" unless signing_time_ok?(now: now)
+      raise VersionError, "unsupported version: #{version}" unless
+        SUPPORTED_VERSIONS.include?(@token['version'])
```

Comment:
I would combine these two lines, rather than having a line break after `unless`, or replace them with an `unless ...\n  ...\n  end` structure.

:bike: :house:

Reply:
It is in the category "do not fear a few long lines", mentioned by @ct-clearhaus earlier.